sklearn/utils/estimator_checks.py

```diff
@@ -269,6 +269,24 @@ def _yield_all_checks(name, estimator):
     yield check_fit_idempotent
 
 
+def _construct_instance(Estimator):
```

Comment:
Isn't this already refactored in master? can you merge master and check out my PR to you?

Reply:
Lol never mind, I messed around with my branches too much, this wasn't in the PR I that was merged.